website/static/js/nodeControl.js

```diff
@@ -144,8 +144,9 @@ var ProjectViewModel = function(data, options) {
                 return {newValue: newValue};
             }
         }));
+    } else {
+      $('#nodeDescriptionEditable').html($osf.markdownAcceptsLinks(self.description()));
```

Comment:
Could you rename this function so that it's just "linkifyText" or similar, since it doesn't do anything with Markdown?